test/cases/ostree-simplified-installer.sh

```diff
@@ -377,42 +378,43 @@ greenprint "🧽 Clean up container blueprint and compose"
 sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
 sudo composer-cli blueprints delete container > /dev/null
 
-############################################################
-##
-## Build edge-simplified-installer image
-##
-############################################################
+### Will move this negative test cases into downstream.
 
 # Verify that composer can report proper error message if no installation device is specified in blueprint
 # https://github.com/osbuild/osbuild-composer/pull/1755
-greenprint "Negative test: checking error message when no installation device specified"
-
-greenprint "📋 Preparing installer blueprint with no installation device"
-tee "$BLUEPRINT_FILE" > /dev/null << EOF
-name = "simplenodevice"
-description = "A rhel-edge simplified-installer image without installation device specified"
-version = "0.0.1"
-modules = []
-groups = []
-EOF
-
-sudo composer-cli blueprints push "$BLUEPRINT_FILE"
-sudo composer-cli blueprints depsolve simplenodevice
-
-result=$(sudo composer-cli compose start-ostree simplenodevice "$INSTALLER_TYPE" --ref "$OSTREE_REF" --url "$PROD_REPO_URL" 2>&1)
-expected='boot ISO image type "edge-simplified-installer" requires specifying an installation device to install to'
-
-echo "Command output is: $result"
-
-greenprint "🎏 Checking if command result contains expected error message."
-if [[ "$result" == *"$expected"* ]]; then
-    greenprint "Success: osbuild-composer can report proper error messages when no installation device specified for simplified installer image"
-else
-    greenprint "Failed: expected error message not found."
-    clean_up
-    exit 1
-fi
-
+# greenprint "Negative test: checking error message when no installation device specified"
+
+# greenprint "📋 Preparing installer blueprint with no installation device"
+# tee "$BLUEPRINT_FILE" > /dev/null << EOF
+# name = "simplenodevice"
+# description = "A rhel-edge simplified-installer image without installation device specified"
+# version = "0.0.1"
+# modules = []
+# groups = []
+# EOF
+
+# sudo composer-cli blueprints push "$BLUEPRINT_FILE"
+# sudo composer-cli blueprints depsolve simplenodevice
+
+# result=$(sudo composer-cli compose start-ostree simplenodevice "$INSTALLER_TYPE" --ref "$OSTREE_REF" --url "$PROD_REPO_URL" 2>&1)
+# expected='boot ISO image type "edge-simplified-installer" requires specifying an installation device to install to'
+
+# echo "Command output is: $result"
+
+# greenprint "🎏 Checking if command result contains expected error message."
+# if [[ "$result" == *"$expected"* ]]; then
+#     greenprint "Success: osbuild-composer can report proper error messages when no installation device specified for simplified installer image"
+# else
+#     greenprint "Failed: expected error message not found."
+#     clean_up
+#     exit 1
+# fi
```

Comment:
Is this commented out and not deleted because it's a temporary move?

Reply:
@yih-redhat can you answer the above?

Reply:
ok I think the answer it's here https://github.com/osbuild/osbuild-composer/pull/2434/files#diff-49189d605ea0886bf92e7dfa5b511edf0735acf56197f5de2945dc1acc60bd8dR381 - so we can lift this conversation for now and let Yi clean it up after this merges